wp-parsely-2.5/src/class-parsely.php

```diff
@@ -1547,15 +1708,16 @@ private function get_categories( $post_id, $delimiter = '/' ) {
 	 * Safely returns options for the plugin by assigning defaults contained in optionDefaults.  As soon as actual
 	 * options are saved, they override the defaults.  This prevents us from having to do a lot of isset() checking
 	 * on variables.
+	 *
+	 * @return array
 	 */
 	private function get_options() {
 		$options = get_option( self::OPTIONS_KEY );
```

Comment:
This can probably be condensed as follows
```suggestion
		$options = get_option( self::OPTIONS_KEY, $this->options_defaults );
```

Reply:
I've fixed it here: https://github.com/Parsely/wp-parsely/pull/382